Makefile

```diff
@@ -61,6 +61,8 @@ install: build
 	install -m 644 ${INSTALLOPTS} etc/renderer/mapserver/fonts.lst         $(DESTDIR)/etc/tirex/renderer/mapserver/fonts.lst
 	install -m 755 ${INSTALLOPTS} -d                                       $(DESTDIR)/etc/tirex/renderer/mapnik
 	install -m 644 ${INSTALLOPTS} etc/renderer/mapnik.conf.dist            $(DESTDIR)/etc/tirex/renderer/mapnik.conf
+        install -m 755 ${INSTALLOPTS} -d                                       $(DESTDIR)/etc/logrotate.d
+        install -m 644 ${INSTALLOPTS} debian/etc/logrotate.d/tirex-master      $(DESTDIR)/etc/logrotate.d/tirex-master
```

Comment:
@xamanu Where do you suggest placing it, so that creating deb packages is still possible? This is your commit which caused it missing: https://github.com/openstreetmap/tirex/commit/ae3b7495083a942b4196e435759054d77721164e

How about moving it to the etc root folder? in parallel to the _renderer_ folder? Would this work with the debian process? It works when calling `make deb` to build local deb packages.

Reply:
> How about moving it to the etc root folder? in parallel to the renderer folder? Would this work with the debian process? It works when calling make deb to build local deb packages.

Yes, that would be great, and is in line what i meant.